### ISSUE TITLE
Differentiate the "docker run" cluster driver type in telemetry

### ISSUE THREAD

**Issue by cbron** [rancher/rancher#30523]
Currently we have no way of knowing how many "docker run" clusters are running as the driver type is k3s.

We can't purely change the driver type in the cluster object, because we use the k3s type to display cluster metrics and options. We could do what RancherD did and add your own type (ClusterDriverRancherD) and include that as a "K3SBased" type. 

An easier alternative is find an easier way to identify that its a docker install, and then add [logic in telemetry](https://github.com/rancher/telemetry/blob/master/collector/cluster.go#L101) to increment that type, which wouldn't actually exist outside of telemetry. 

If we can get this in 2.5.5 that'd be great, if it slips to 2.6 its not a huge problem.

**Comment by rawmind0** [comment]
Submitted PR https://github.com/rancher/telemetry/pull/46  to address the issue. 

Added logic to telemetry client to differentiate when Rancher is deployed with `docker run`, incrementing `k3sBased` label on telemetry

**Comment by rawmind0** [comment]
PR https://github.com/rancher/telemetry/pull/46 merged and cut a new telemetry version, [v0.5.13](https://github.com/rancher/telemetry/releases/tag/v0.5.13)

How to test it:
* Deploy Rancher v2.5.5 using [single-node-docker](https://rancher.com/docs/rancher/v2.x/en/installation/other-installation-methods/single-node-docker/) installation
* Get telemetry client cmd from the Rancher docker, `docker exec -i <RANCHER_DOCKER> ps -ef | grep telemetry` 
* Download new telemetry binary from https://github.com/rancher/telemetry/releases/tag/v0.5.13
* Check new telemetry client `cluster.driver` output (should be `k3sBased` instead of `k3s`):
`telemetry client --url https://<RANCHER_URL>/v3 --token-key <TELEMETRY_TOKEN> --once`

```
...
  "cluster": {
    "driver": {
      "k3sBased": 1
    },
...
```

**Comment by aaronRancher** [comment]
This feature is verified with Rancher server `rancher\rancher:v2.5.5`

Steps:

- Deploy Rancher using single-node-docker installation
- Get telemetry client cmd from the Rancher docker, `docker exec -i <RANCHER_DOCKER> ps -ef | grep telemetry`
- Download new telemetry binary from https://github.com/rancher/telemetry/releases/tag/v0.5.13
- Check new telemetry client `cluster.driver` output (should be `k3sBased` instead of `k3s`): `telemetry client --url https://<RANCHER_URL>/v3 --token-key <TELEMETRY_TOKEN> --once`

Result:
Telemetry client `cluster.driver` output. is `k3sBased`
```
  "cluster": {
    "active": 1,
    "total": 1,
    ...
    "driver": {
      "k3sBased": 1
    },
   ...
  },
```
